python/test/unit/function/test_function.py

```diff
@@ -361,3 +367,65 @@ def test_interpolation_function(mesh):
     uh = Function(Vh)
     uh.interpolate(u)
     assert np.allclose(uh.vector.array, 1)
+
+
+@pytest.mark.parametrize("ct, tdim", [(cpp.mesh.CellType.triangle, 2),
+                                      (cpp.mesh.CellType.quadrilateral, 2),
+                                      (cpp.mesh.CellType.tetrahedron, 3),
+                                      (cpp.mesh.CellType.hexahedron, 3)])
+def test_eval_parallel(ct, tdim):
+    if tdim == 2:
+        mesh = UnitSquareMesh(MPI.COMM_WORLD, 7, 7, cell_type=ct)
+    else:
+        mesh = UnitCubeMesh(MPI.COMM_WORLD, 7, 7, 7, cell_type=ct)
+    V = FunctionSpace(mesh, ("CG", 1))
+
+    def func(x):
```

Comment:
Maybe use lambda function in `interpolate`?

Reply:
Since the function is reused in to obtain the exact solution, I do not think we should use a lambda function.